### PR TITLE
PCHR-2035: Dropdown styles fixed

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR variables are used to define `height` and `border-color` for Custom Select and UI select.

## Technical Details
`$input-height-base`and `$input-border` is used for `height` and `border-color` in shoreditch, so that when the same partials are imported into SSP(https://github.com/compucorp/civihr-employee-portal-theme/pull/173) the values change according to SSP's variables.

### CODE REVIEW

File: scss/bootstrap/overrides/style/_ui-select.scss [hunk 1]
```diff
@@ -59,12 +59,18 @@
 
 .ui-select-container:not([theme="civihr-ui-select"]) {
   &:not(.ui-select-multiple) {
-    .btn:hover {
-      background: #FFF;
+
+    .btn{
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_ui-select.scss]: missing space before `{`

File: scss/bootstrap/components/_custom-select.scss [hunk 2]
```diff
@@ -11,8 +11,9 @@ $crm-customselect-feedback-padding-right: $crm-customselect-padding-right + 10;
 
   > select {
     background: transparent;
-    border: 1px solid #949caa;
-    padding: 6px #{$crm-customselect-padding-right}px 6px 12px;
+    border: 1px solid $input-border;
+    height: $input-height-base;
```
Comment (AkA84) [inline scss/bootstrap/components/_custom-select.scss:15]: the `height` wasn't specified before, why do we need to specify it now?

File: scss/bootstrap/overrides/style/_ui-select.scss [hunk 3]
```diff
@@ -59,12 +59,18 @@
...
+      height: $input-height-base;
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_ui-select.scss]: > the `height` wasn't specified before, why do we need to specify it now?

same here

File: scss/bootstrap/overrides/style/_ui-select.scss [hunk 4]
```diff
@@ -90,17 +96,18 @@
     }
 
     input {
-      border: 1px solid #727E8A !important;
+      border: 1px solid $input-border !important;
       box-shadow: none !important;
+      height: $input-height-base;
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_ui-select.scss:101]: > the `height` wasn't specified before, why do we need to specify it now?

same here

File: scss/bootstrap/overrides/style/_ui-select.scss [hunk 5]
```diff
@@ -116,8 +123,9 @@
 }
 
 .ui-select-multiple:not([theme="civihr-ui-select"]) {
-  border: 1px solid #949caa;
+  border: 1px solid $input-border;
   padding: 3px !important;
+  height: $input-height-base;
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_ui-select.scss:128]: > the `height` wasn't specified before, why do we need to specify it now?

same here

File: scss/bootstrap/overrides/style/_ui-select.scss [hunk 6]
```diff
@@ -90,17 +96,18 @@
...
     }
 
     &:after {
       content: '\f002';
       bottom: 0;
       font-family: "FontAwesome";
       display: inline-block;
-      height: 33px;
-      line-height: 35px;
+      height: 100%;
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_ui-select.scss:109]: why the change from a fixed height to a fluid height?

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: scss/bootstrap/overrides/style/_ui-select.scss (hunk 6 above)
Comment (deb1990) [reply scss/bootstrap/overrides/style/_ui-select.scss:109]: If I have a fixed height, everytime I need to change it if I change the height of the dropdown, but 100% works regardless

File: scss/bootstrap/components/_custom-select.scss (hunk 2 above)
Comment (deb1990) [reply scss/bootstrap/components/_custom-select.scss:15]: This is done to keep a consistent look across the theme.
`$input-height-base` variable is present for this purpose only and used by `.form-control` class which works with `input`, as this is a `select` adding the height does the same thing.
Previously the height of the select was calculated based on padding, but specifying the height is more reliable.

File: scss/bootstrap/overrides/style/_ui-select.scss (hunk 5 above)
Comment (deb1990) [reply scss/bootstrap/overrides/style/_ui-select.scss:128]: Commented in original review point

File: scss/bootstrap/overrides/style/_ui-select.scss (hunk 4 above)
Comment (deb1990) [reply scss/bootstrap/overrides/style/_ui-select.scss:101]: Commented in original review point

File: scss/bootstrap/overrides/style/_ui-select.scss (hunk 3 above)
Comment (deb1990) [reply scss/bootstrap/overrides/style/_ui-select.scss]: Commented in original review point

**Review by AkA84 — APPROVED** [review]
